ERCS/ERC-chain_id_rpc.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Chain-ID-Based RPC Provider URLs
+description: Standardizes how RPC providers expose node endpoints by chain-ID to simplify multichain integration.
+author: Soham Zemse (@zemse)
+discussions-to: <URL>
+status: Draft
+type: Meta
```

Comment:
Why do you think this is a Meta EIP?

Reply:
I just saw some other Meta eips and then this proposal would be out of place there. It should be an ERC right?